.github/workflows/build.yml

```diff
@@ -1366,9 +1261,8 @@ jobs:
 
           if [ "$PRERELEASE" == "false" ]
           then
-            # The order here matters, because the Chinese build has a slightly higher version code due to the suffix of 1 vs 0
             ./pico-cli upload-build --app-id $PICO_APP_ID --app-secret $PICO_APP_SECRET --region noncn --apk OpenBrush_Pico_$VERSION.apk --channel 2 --notes-en "Version $VERSION" --device 'PICO Neo3,PICO Neo3 Pro,PICO Neo3 Eye,PICO 4,PICO 4 Ultra,PICO 4 Pro'
-            ./pico-cli upload-build --app-id $PICO_APP_ID --app-secret $PICO_APP_SECRET --region noncn --apk OpenBrush_Pico_CN_$VERSION.apk --channel 1 --notes-en "Version $VERSION" --device 'PICO Neo3,PICO Neo3 Pro,PICO Neo3 Eye,PICO 4,PICO 4 Ultra,PICO 4 Pro'
+            ./pico-cli upload-build --app-id $PICO_APP_ID --app-secret $PICO_APP_SECRET --region cn --apk OpenBrush_Pico_$VERSION.apk --channel 1 --notes-en "Version $VERSION" --device 'PICO Neo3,PICO Neo3 Pro,PICO Neo3 Eye,PICO 4,PICO 4 Ultra,PICO 4 Pro'
```

Comment:
Is this correct?

Reply:
we really need to make sure we understand how this stuff works. Channel 1 is the China Store, Channel 2 is non-China, but there's also region, and... I don't know

Reply:
I'm not sure. Since we're outside of China, I think we need to use --region noncn, even when we upload to --channel 1 (and certainly for --channel 2)

Reply:
Oh, I get you. It'll be doing something lime be changing the endpoint target for the upload, that wasn't clear.